src/main/java/hudson/plugins/parameterizedtrigger/BlockableBuildTriggerConfig.java

```diff
@@ -30,7 +30,6 @@
  */
 public class BlockableBuildTriggerConfig extends BuildTriggerConfig {
     private final BlockingBehaviour block;
-    public boolean buildAllNodesWithLabel;
```

Comment:
I'll drop this change as I not not want to mix this PR of mine with code deprecation.